app/src/main/java/org/fossasia/openevent/app/core/auth/login/LoginFragment.java

```diff
@@ -70,13 +69,24 @@ public void onStart() {
             if (!validator.validate())
                 return;
 
+            loginFragmentViewModel.getLogin().setEmail(binding.emailDropdown.getText().toString());
+            loginFragmentViewModel.getLogin().setPassword(binding.userPassword.getText().toString());
+
             ViewUtils.hideKeyboard(view);
             loginFragmentViewModel.login();
+            loginFragmentViewModel.encryption();
```

Comment:
It's the responsibility of VM to do whatever is required and not rely on view to tell what to do, this call is never done without login call, so should be implicitly coupled with it